src/components/DayOverview/DayOverviewPdf.tsx

```diff
@@ -0,0 +1,202 @@
+    header_collection: { flex: 4, fontFamily: "Helvetica-Bold", ...generalCellStyle },
+    header_six: { flex: 10, fontFamily: "Helvetica-Bold", ...generalCellStyle },
+
+    // cells
+    cell_logo: { flex: 1, ...generalCellStyle },
+    cell_name: { flex: 4, ...generalCellStyle },
+    cell_postcode: { flex: 4, ...generalCellStyle },
+    cell_time: { flex: 2, ...generalCellStyle },
+    cell_collection: { flex: 4, ...generalCellStyle },
+    cell_instructions: { flex: 10, ...generalCellStyle },
+});
+
+const dateTimeToAMPM = (datetime: string): string => {
+    return new Date(datetime).getHours() <= 11 ? "AM" : "PM";
+};
+
+const CustomSVG: React.FC<CustomSVGProps> = ({ icon, color, fill }) => (
+    <Svg width={12} height={12} viewBox="0 0 512 512" style={styles.svg}>
+        <Path
+            fill={fill ? color : undefined}
+            stroke={color}
+            strokeWidth={40}
+            d={icon.icon[4] as string}
+        />
+        {/*IconDefinition[4] is defined as string | string[] but in reality there is no use case*/}
+        {/*with string[] as svgPathData*/}
+    </Svg>
+);
+
+const DayOverviewerDate: React.FC<{ date: Date }> = ({ date }) => {
+    const dateString = date.toLocaleDateString("sv-SE");
+    return (
+        <View style={styles.row}>
+            <Text style={styles.bold}>Date: </Text> <Text>{dateString}</Text>
+        </View>
+    );
+};
+
+const DayOverviewerLocation: React.FC<{ location: string }> = ({ location }) => (
+    <View style={styles.row}>
+        <Text style={styles.bold}>Location: </Text> <Text>{location}</Text>
+    </View>
+);
+const DayOverviewerHeader: React.FC<{}> = () => {
+    return (
+        <View style={styles.row}>
+            <View style={styles.header_flag}></View>
+            <View style={styles.header_name}>
+                <Text style={styles.bold}>Name</Text>
+            </View>
+            <View style={styles.header_postcode}>
+                <Text style={styles.bold}>Postcode</Text>
+            </View>
+            <View style={styles.header_time}>
+                <Text style={styles.bold}>Time</Text>
+            </View>
+            <View style={styles.header_six}>
+                <Text style={styles.bold}>Instructions</Text>
+            </View>
+        </View>
+    );
+};
+
+// Database return type has null
+// But we know a client foreign key must map to an existing client
+// And collection_date is specified to be within a range
+// Hence both cannot be null
+const DayOverviewerRow: React.FC<{ row: ParcelOfSpecificDateLocation }> = ({ row }) => {
+    return (
+        <View style={styles.row}>
+            <View style={styles.cell_logo}>
+                <View style={styles.row}>
+                    <CustomSVG icon={faSquare} color="black" fill={false} />
+                    {row.clients!.flagged_for_attention ? (
+                        <CustomSVG icon={faFlag} color="orange" fill={true} />
+                    ) : (
+                        <></>
```

Comment:
do you want the empty fragment there (i.e. a "gap" sort of if you're not rendering the icon)? if not, can you make this conditional rendering (`&&`)?

Reply:
It doesn't matter as the column width should be sufficient for two icons - but using `&&` seems tidier so refactored